auth0/src/test/java/com/auth0/android/authentication/AuthenticationAPIClientTest.java

```diff
@@ -421,6 +421,51 @@ public void shouldLoginWithOAuthAccessTokenSync() throws Exception {
         assertThat(credentials, is(notNullValue()));
     }
 
+    @Test
+    public void shouldLoginWithNativeSocialToken() throws Exception {
+        mockAPI
+                .willReturnSuccessfulLogin();
```

Comment:
The formatting is a little funky here - is this common?

If not I think this would read better all on one line, like it is below.

Reply:
Good catch, resolved!